contracts/MergedEquippable/README.md

```diff
@@ -761,6 +761,7 @@ async function mintTokens(
   gem: SimpleEquippable
 ): Promise<void> {
   console.log("Minting tokens");
+  const [ , owner] = await ethers.getSigners();
```

Comment:
The next line should be replaced by this one. The script only has this line, but it seems I just added this line instead of replacing the line above 🙈 

The idea is to assign the owner the second available signer. This way the owner still has to accept the child and asset. Using the default (first) signer kinda defeats half of most user journeys now 😅